src/components/ProductListing/Products.jsx

```diff
@@ -1,4 +1,5 @@
-import { Header, Footer, Main } from "./index";
+import { Header, Main } from "./index";
+import { Footer } from "../Footer";
```

Comment:
Instead of importing your Header and Footer on every page why not render it in App.js regardless of the component/ state if you are using it on all components?

Reply:
I didn't know we can do that, Thanks, will look into this